ui/app/styles/components/box-label.scss

```diff
@@ -63,6 +63,6 @@ label.box-label {
   color: $grey;
 
   .is-selected & {
```

Comment:
we no longer have a var called `$text`. This was a very very confusing var that was overrode in several places. We never used it consistently and so I replaced it with the matching color var.